server/chat-plugins/modlog-viewer.ts

```diff
@@ -451,17 +486,29 @@ export const commands: ChatCommands = {
 		void getModlog(
 			connection,
 			roomid,
-			target,
+			search,
+			target.replace(/,?\s*(room|lines)\s*=[^,]*,?/g, ''),
```

Comment:
What is this?

Reply:
this is because the `getModlog()` function re-adds room= and line= qualifiers, so we remove them from the query to avoid duplicating them